File: lecture_notes/07 basics/1-identifiers.js
'use strict';

const MAX_PURCHASE = 2000;

const calculateTotal = (prices) => {
  let amount = 0;
  for (const price of prices) {
    amount += price;
  }
  return amount;
};

// Usage

const purchase = [1500, 100, 10, 50];
const total = calculateTotal(purchase);
if (total <= MAX_PURCHASE) {
  console.log({ total });
}

File: lecture_notes/07 basics/1-identifiers.py
# Implementation

MAX_PURCHASE = 2000

def calculateTotal(prices):
  amount = 0
  for i in prices:
    amount = amount + i
  return amount

# Usage

purchase = [1500, 100, 10, 50]
total = calculateTotal(purchase)
if total <= MAX_PURCHASE:
  print(total)
